.github/workflows/jules-pyflakes-optimizer.yml

```
@@ -22,7 +22,7 @@ jobs:
       - name: Set up Python Environment
         uses: actions/setup-python@v5
         with:
-          python-level: '3.11'
+          python-level: "3.11"
```

actionlint (1.7.12)

[error] input "python-level" is not defined in action "actions/setup-python@v5". available inputs are "allow-prereleases", "architecture", "cache", "cache-dependency-path", "check-latest", "freethreaded", "python-version", "python-version-file", "token", "update-environment"

(action)